facade/service.go

```diff
@@ -64,17 +65,19 @@ var (
 
 // AddService adds a service; return error if service already exists
 func (f *Facade) AddService(ctx datastore.Context, svc service.Service) (err error) {
-	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.AddService"))
 	var tenantID string
+	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.AddService"))
+	alog := f.auditLogger.Action("Add").Message(ctx, "Adding new Service: " + svc.Name).ID(svc.GetID()).Type(svc.GetType())
```

Comment:
We should try to avoid embedding data in the message string. Please change the message string to "Adding new service" and then include `WithField("servicename", svc.Name)`